third-culture-software/bhima
# Improve the stock entry report/receipt

Make the stock entry report a little clearer.

Closes https://github.com/IMA-WorldHealth/bhima/issues/6531
Closes https://github.com/IMA-WorldHealth/bhima/issues/6565

**TESTING**
- Use bhima_test
- Create a shipment with a few items, and mark it ready to ship
- Do the stock exit for the shipment.
- Do a partial stock entry for the shipment (reduce the count on one of the items).
  - Examine the resulting report (you can also access this via the Stock Movements registry)
    - Notice the shipment status
    - Notice the columns for partial deliveries are shown
- Complete the stock entry for the shipment
  - Examine the resulting report
    - Notice that the shipment status is now marked complete
    - Notice the columns related to partial deliveries are not shown since the order is complete
 - Create a new shipment
    - mark it complete
    - do the stock exit for it
    - do the stock entry for it (accept it without changing quantities so it is completed)
    - Review the resulting report
- Go to Stock Exit and change to "Depot Secondaire"
   - Click on [Transfer] and select the requisition for SM.8.2
   - The resulting report does not include info about the Shipment since the old bhima_test data for that requisition was created before shipments were automatically created when doing stock Exits.
   - Go to the Stock Movements page and show the "document' for it and review it
